.github/workflows/markdown-link-check.yml

```diff
@@ -0,0 +1,18 @@
+name: Check Markdown Links
+
+on: push
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    env:
+      ZENML_DEBUG: 1
+      ZENML_ANALYTICS_OPT_IN: false
+    steps:
+    - uses: actions/checkout@master
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        use-quiet-mode: 'no'
+        use-verbose-mode: 'no'
```

Comment:
Not sure what this looks like when it runs, but is it worth keeping the verbose mode in? Some of our links are redirects, and I'm not sure if those fail or not?

Reply:
This is my latest run. We can change that if its needed. But I find a little clutter with the verbose mode.

https://github.com/dnth/zenml/runs/7726267706?check_suite_focus=true

Reply:
Yeah I guess let's start with just the non-verbose mode and if we need more info we can always add it back in?

Reply:
Sure. I'm open to changes to use other tools too in case this is a little cumbersome that the checks only happens on Github